docs/cds/cds.rst

```diff
@@ -0,0 +1,452 @@
+.. doctest-skip-all
+
+.. _astroquery.cds:
+
+**********************************
+CDS MOC Service (`astroquery.cds`)
+**********************************
+
+Getting started
+===============
+
+This module provides a python interface for querying the `CDS MOCServer`_.
+
+MOC is an `IVOA standard`_ enabling description of arbitrary sky regions. Based on the HEALPix sky tessellation, it maps
+regions on the sky into hierarchically grouped predefined cells. It corresponds to a set of HEALPix cells at different
+orders.
+
+For those wanting to know more about MOCs, please refer to this `IVOA paper
+<http://ivoa.net/documents/MOC/20140602/REC-MOC-1.0-20140602.pdf>`_ and the `MOCPy's documentation
+<https://github.com/cds-astro/mocpy>`_ developed by the CDS.
+
+CDS has set up a server known as the `MOCServer <http://alasky.unistra.fr/MocServer/query?>`_ storing data-set names
+each associated with a MOC spatial coverage and some meta-datas giving a more detailed explanation of the data-set.
+
+The MOCServer aims at returning the data-sets having at least one source lying in a specific sky region defined by the
+user. Internally the MOCServer performs the intersection between the given sky region and the MOCs associated with each
+data-sets. Because the MOC associated to a data-set describes its sky coverage, if the above intersection is not null
+then the MOCServer knows that some sources of this data-set are in the user defined sky region.
+
+To be aware of what the MOCServer returns, please refers to this `link
+<http://alasky.unistra.fr/MocServer/query?RA=10.8&DEC=32.2&SR=1.5&intersect=overlaps&get=record&fmt=html>`_.
+We have queried the MOCServer with a cone region of center ra, dec = (10.8, 32.2) deg and radius = 1.5 deg. In return,
+the MOCServer gives a list of data-sets each tagged with an unique ID along with some other meta-datas too e.g.
+``obs_title``, ``obs_description``, ``moc_access_url`` (url for accessing the MOC associated with the data-set. Usually
+a FITS file storing a list of HEALPix cells).
+
+It is also possible to ask the MOCServer for retrieving data-sets based on their meta-data values. `Here
+<http://alasky.unistra.fr/MocServer/query?RA=10.8&DEC=32.2&SR=1.5&intersect=overlaps&get=record&fmt=html&expr=(dataproduct_type=image)>`_
+we have queried the MOCServer for only the image data-sets being in the cone defined above (``dataproduct_type``
+meta-data equals to ``"image"``).
+
+This package implements two methods:
+
+* :meth:`~astroquery.cds.CdsClass.query_region` retrieving data-sets (their associated MOCs and meta-datas) having sources in a given region.
+* :meth:`~astroquery.cds.CdsClass.find_data_sets` retrieving data-sets (their associated MOCs and meta-datas) based on the
+  values of their meta-datas.
+
+Requirements
+----------------------------------------------------
+The following packages are required for the use of this module:
+
+* `astropy-healpix`_
```

Comment:
could you please also add these optional dependencies to the main list in `docs/index.rst`?